refactor: LEAP-1296: remove FF_LOPS_86 feature flag

Gondragos opened the pull request:
Removed the `fflag_feat_front_lops_86_datasets_storage_edit_short` feature flag and its associated code logic for dataset storage editability.

| **Key** | `fflag_feat_front_lops_86_datasets_storage_edit_short` |
|---|---|
| **Variable Name** | `FF_LOPS_86` |
| **Ends up as** |  🟥 `False` |
| **Description** | Adding the ability to toggle dataset storage editability. |

<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

Gondragos committed:
refactor: remove FF_LOPS_86 feature flag

Removed the `fflag_feat_front_lops_86_datasets_storage_edit_short` feature flag and its associated code logic for dataset storage editability.

nass600 commented on web/libs/datamanager/src/components/MainView/DataView/Table.jsx:
```diff
@@ -142,20 +142,7 @@ export const DataView = injector(
-                  <Elem name="text">Check your storage settings and resync to import records</Elem>
-                  <Button
-                    onClick={async () => {
-                      window.open("./settings/storage");
```
Are we removing the rest of the component tree and the link to settings for any specific reason?

Gondragos replied:
I'm not sure I understand the question, but here I'm removing only the part that already doesn't appear in UI with the feature flag's state we have at the end of that feature flag's lifecycle (isFF(FF_LOPS_86) === `False`).

nass600 replied:
Never mind, I checked it wrongly

Gondragos commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13900681969)
> Successfully merged:  create mode 100644 web/libs/editor/tests/integration/e2e/control_tags/visibility.cy.ts

Gondragos committed:
Merge branch 'develop' into 'fb-LEAP-1296/stale-ff'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13900681969

Gondragos committed:
refactor: remove stale feature flag

Removed 'fflag_feat_front_lops_86_datasets_storage_edit_short' from the stale feature flags list as it is no longer in use.

Gondragos committed:
Merge branch 'develop' into fb-LEAP-1296/stale-ff

# Conflicts:
#	label_studio/core/feature_flags/stale_feature_flags.py
#	web/libs/datamanager/src/components/MainView/DataView/Table.jsx
#	web/libs/datamanager/src/utils/feature-flags.js
